tests/providers/gcp/gcp_managed_relational_db_test.py

```diff
@@ -210,10 +212,12 @@ def testValidateMachineType(self):
     with self._PatchCriticalObjects():
       db = self.createManagedDbFromSpec(self.createPostgresSpecDict())
       self.assertRaises(ValueError, db._ValidateMachineType, 0, 0)
-      db._ValidateMachineType('3840MiB', 1)
-      db._ValidateMachineType('5376MB', 4)
-#       db._ValidateMachineType(db.spec.vm_spec.machine_type.memory,
-#                               db.spec.vm_spec.machine_type.cpus)
+      self.assertRaises(ValueError, db._ValidateMachineType, 3840, 0)
```

Comment:
Add a test for the case where the user forgets to specify CPU and/or memory, and make sure we return proper error message.